src/Map/src/Elements.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map;
+
+/**
+ * Represents a collection of map elements.
+ *
+ * @author Sylvain Blondeau <contact@sylvainblondeau.dev>
+ */
+abstract class Elements
+{
+    protected \SplObjectStorage $elements;
+
+    public function __construct(
+        array $elements,
+    ) {
+        $this->elements = new \SplObjectStorage($elements);
```

Comment:
AFAIK `SplObjectStorage` does not have constructor arguments, so using `$elements` has no effect here